### PR TITLE
configuración CI v1

### PR DESCRIPTION
**Pull request by JuanManu59**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+      - name: Code style
+        run: npm run lint
+
+      - name: UI E2E Tests
+        uses: cypress-io/github-action@v4
+        with:
+          browser: chrome
+          headless: true
+
```
Comment (NicolasB2) [inline .github/workflows/main.yml]: EOF

Comment (JuanManu59) [reply .github/workflows/main.yml]: resolved

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+16.15.1
```
Comment (AlejaGonzalezV) [inline .nvmrc]: EOF

Comment (JuanManu59) [reply .nvmrc]: resolved

**Review by AlejaGonzalezV — APPROVED** [review]

**Pull request merged**